lib/pharos/phases/configure_service_account.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    class ConfigureServiceAccount < Pharos::Phase
+      title "Configure 'pharos-admin' service account"
+
+      ADMIN_USER = 'pharos-admin'
+      KUBECONFIG_PARAM = '--kubeconfig=/etc/kubernetes/admin.conf'
+
+      def call
+        create_service_account
+        create_cluster_role_binding
+
+        config = build_config
+
+        if config_file.exist?
+          existing_config = Pharos::Kube::Config.new(config_file.read)
+          config << existing_config
+        end
+
+        config_file.write(config.dump, overwrite: true)
+        config_file.chmod('0600')
+
+        validate
+      end
+
+      def validate
+        transport.exec!('kubectl get -n kube-system serviceaccount/pharos-admin')
```

Comment:
We are using `kubectl` here because client is not yet configured?

Reply:
I added that just to validate that `kubectl` without sudo on master works without `KUBECONFIG=` or `--kubeconfig=`.

The next phase actually probably should to be changed to use the file from home instead of `/etc/kubernetes`